packages/aws-cdk-lib/custom-resources/lib/aws-custom-resource/aws-custom-resource.ts

```diff
@@ -370,7 +370,7 @@ export interface AwsCustomResourceProps {
   readonly logGroup?: logs.ILogGroup;
 
   /**
-   * Whether to install the latest AWS SDK v2.
+   * Whether to install the latest AWS SDK.
```

Comment:
While it would be acceptable to specify "v3", I decided not to make any specific reference to the major version, as this would require updating the code each time the major version changes.

Reply:
Please update this comment to match that it is handler-dependent; see https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/custom-resource-handlers/lib/custom-resources/aws-custom-resource-handler/aws-sdk-v3-handler.ts#L40 and https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/custom-resource-handlers/lib/custom-resources/aws-custom-resource-handler/aws-sdk-v2-handler.ts#L64.